crates/router/src/routes/app.rs

```diff
@@ -89,9 +89,9 @@ impl AppState {
             store,
             conf,
             #[cfg(feature = "email")]
-            email_client,
+            email_client: Some(email_client),
```

Comment:
@Chethan-rao We are making this field optional as it is not required for other services, please confirm if this is ok

Reply:
Hey @jagan-jaya , I assume you are using your own Appstate why does the core application AppState need to be changed...